assets/styles/components/_breadcrumb.scss

```diff
@@ -0,0 +1,21 @@
+@use "../base/variables";
+
+.breadcrumb {
+  color: variables.$color-grey-120;
+  text-transform: uppercase;
+
+  &__link {
+    text-decoration: none;
+    color: variables.$color-grey-120;
+  }
+
+  &__tail {
+    color: variables.$color-grey-100;
+  }
+
+  & > *:not(:last-child)::after {
+    display: inline-block;
+    margin: 0 5px;
```

Comment:
I propose again to use the multi-line form with `margin-top` and `margin-right`. It's not very important, but it's for harmonization and readability.